setup.py

```diff
@@ -1,4 +1,4 @@
 from setuptools import setup
 
 # Name and version specified here for `sdist` only.
-setup(name='edgePy', version='0.0.1')
+setup(name='edgePy', version='0.0.1', include_package_data=True)
```

Comment:
Redundant add. This option is already turned on in `setup.cfg`:

https://github.com/r-bioinformatics/edgePy/blob/cf1691a2c48b7753e3bbd0d44981e196fe8cb7bf/setup.cfg#L24